### PR TITLE
Add es-ui-upgrades

### PR DESCRIPTION
**Pull request by lumbar527**

### CODE REVIEW

File: manifests/es-ui-upgrades.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: ES-ui-upgrades
+authors: Roger Tabris
+homepage: https://github.com/lumbar527/es-ui-upgrades
+license: GPL-3-or-later
+version: 7d06da8e50be2986fb5814880941ec2e1c62f5f2
+shortDescription: A plugin that aims to transform the UI.
+description: This plugin aim to be the first plugin to transform the UI. It tries to make it into a more digital interface, with floating transparent blue windows.
+url: https://github.com/lumbar527/es-ui-upgrades/archive/7d06da8e50be2986fb5814880941ec2e1c62f5f2
```
Comment (lumbar527) [inline manifests/es-ui-upgrades.yaml:8]:
```suggestion
url: https://github.com/lumbar527/es-ui-upgrades/archive/7d06da8e50be2986fb5814880941ec2e1c62f5f2.zip
```

File: manifests/es-ui-upgrades.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: ES-ui-upgrades
+authors: Roger Tabris
+homepage: https://github.com/lumbar527/es-ui-upgrades
+license: GPL-3-or-later
```
Comment (MCOfficer) [inline manifests/es-ui-upgrades.yaml:4]:
```suggestion
license: GPL-3.0-or-later
```
https://spdx.org/licenses/GPL-3.0-or-later.html

File: manifests/es-ui-upgrades.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+version: 7d06da8e50be2986fb5814880941ec2e1c62f5f2
+shortDescription: A plugin that aims to transform the UI.
+description: This plugin aim to be the first plugin to transform the UI. It tries to make it into a more digital interface, with floating transparent blue windows.
```
Comment (MCOfficer) [inline manifests/es-ui-upgrades.yaml:7]:
```suggestion
description: This plugin aims to be the first plugin to transform the UI. It tries to make it into a more digital interface, with floating transparent blue windows.
```

File: manifests/es-ui-upgrades.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+url: https://github.com/lumbar527/es-ui-upgrades/archive/7d06da8e50be2986fb5814880941ec2e1c62f5f2.zip
+iconUrl: https://github.com/lumbar527/es-ui-upgrades/raw/7d06da8e50be2986fb5814880941ec2e1c62f5f2/icon.png
+autoupdate:
+  type: commit
+  url: https://github.com/lumbar527/es-ui-upgrades/archive/$version
```
Comment (MCOfficer) [inline manifests/es-ui-upgrades.yaml:12]:
```suggestion
  url: https://github.com/lumbar527/es-ui-upgrades/archive/$version.zip
```

**Review by MCOfficer — CHANGES_REQUESTED** [review]

**Pull request merged**